src/pages/en/guides/content-collections.mdx

```diff
@@ -187,290 +217,211 @@ defineCollection({
+        {blogPostEntry.data.publishedDate.toDateString()}
       </time>
     </li>
   ))}
 </ul>
 ```
 
-### Collection entry types
+### Passing content as props
 
-If a page or component uses content from a `getCollection()` or `getEntry()` query, you can use the `CollectionEntry` utility to type its props:
+A component can also pass an entire content entry as a prop. 
+
+If you do this, you can use the [`CollectionEntry`](/en/reference/api-reference/#collection-entry-type) utility to correctly type your components props using TypeScript.  This utility takes a string argument that matches the name of your collection schema, and will inherit all of the properties of that collection's schema.
 
 ```astro /CollectionEntry(?:<.+>)?/
 ---
 // src/components/BlogCard.astro
 import type { CollectionEntry } from 'astro:content';
-
 interface Props {
-  // Get type of a `blog` collection entry
   post: CollectionEntry<'blog'>;
 }
 
-// `post.data` will match your collection schema
+// `post` will match your 'blog' collection schema type
 const { post } = Astro.props;
 ---
 ```
 
-## Rendering entry content
+### Rendering content to HTML
 
-Every collection entry includes a `render()` function that gives you access to the contents of the Markdown or MDX file. This includes a `<Content />` component for rendering the document body, as well as the document headings and frontmatter modified via remark or rehype.
+Once queried, you can render a collection entry to HTML using the entry `render()` function property. Calling this function gives you access to rendered content and metadata, including both a `<Content />` component and a list of all rendered headings.
 
 ```astro {5}
 ---
 // src/pages/render-example.astro
-import { getEntry } from 'astro:content';
-const entry = await getEntry('blog', 'post-1.md');
-const { Content, headings, remarkPluginFrontmatter } = await entry.render();
+import { getEntryBySlug } from 'astro:content';
+const entry = await getEntryBySlug('blog', 'post-1');
+const { Content, headings } = await entry.render();
 ---
+<p>Written by: {entry.data.author}</p>
+<Content />
 ```
 
-### Access the `<Content />` component from `render()`
 
-To render the content of a Markdown or MDX entry, use the `<Content />` component returned by its `render()` function. This allows you to generate pages from your content entries (see [Generating pages from content collections](#generating-pages-from-content-collections)), add post previews to your homepage, or display your content elsewhere on your site.
+## Generating Routes from Content
 
-For example, this page renders the contents of `content/announcements/welcome.md` and uses some of its frontmatter properties:
+Because content collections live outside of the `src/pages/` directory, its up to you to create [dynamic routes](/en/core-concepts/routing/#dynamic-routes) that will render your content and any collection entries to HTML.
```

Comment:
```suggestion
Because content collections live outside of the `src/pages/` directory, it's up to you to create [dynamic routes](/en/core-concepts/routing/#dynamic-routes) that will render your content and any collection entries to HTML.
```
nit: apostrohe